pkg/proxy/ipvs/netlink_unsupported.go

```diff
@@ -0,0 +1,41 @@
+// +build !linux
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ipvs
+
+import (
+	"fmt"
+)
+
+type emptyHandle struct {
+}
+
+// NewNetLinkHandle will create an EmptyHandle
+func NewNetLinkHandle() NetLinkHandle {
+	return &emptyHandle{}
+}
+
+// EnsureAddressBind checks if address is bound to the interface and, if not, binds it. If the address is already bound, return true.
+func (h *emptyHandle) EnsureAddressBind(address, devName string) (exist bool, err error) {
+	return false, fmt.Errorf("netlink not supported for this platform")
```

Comment:
I believe this fake function should return `{true, nil}` or `{false, nil}`.

IPVS doesn't work on !linux platform, that's true. However, we may run UTs on !linux systems when developing. So, I would like this function and below return err=nil.